apps/trading/components/welcome-dialog/telemetry-approval.tsx

```diff
@@ -0,0 +1,30 @@
+import { Checkbox } from '@vegaprotocol/ui-toolkit';
+import { t } from '@vegaprotocol/i18n';
+import { useTelemetryApproval } from '../../lib/hooks/use-telemetry-approval';
+
+export const TelemetryApproval = () => {
+  const [isApproved, setIsApproved] = useTelemetryApproval();
```

Comment:
chore (blocking): would be good to add unit tests for

- the TelemetryApproval component
- the useTelemetryApproval hook

Reply:
done as well